classes/person.rb

```diff
@@ -1,13 +1,17 @@
 require('securerandom')
-class Person
+require_relative('nameable')
+
+class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = SecureRandom.uuid
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+    super()
   end
 
```

Comment:
@wacoo Great work on adding associations, however, you forgot to add the method for adding rentals that takes the person itself, book, and, date to create a rental, this will help accomplish the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental

Reply:
- [ ] Good job adding the method, but you are missing an important part, this method should take a parameter `date` and `book`, and then create a new instance of rental class like `Rental.new(date, self, book)` with this a has many relationship is created with rentals class